Manifold_6_13_CLEAN_SUMMED.py.ipynb

```diff
@@ -15,21 +15,6 @@
     }
    ],
    "source": [
-    "# Import packages\n",
```

Comment:
woah we shouldn't remove these import statemetnts, the packages are still being used!

Reply:
you're right.